components/classes/b2b/account-history.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/account-history",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "XDM Business Account History",
+  "type": "object",
+  "meta:tags": {
+    "b2bSchema": true
+  },
+  "auditable": true,
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/time-series"],
+  "description": "XDM Business Account History is a standard Experience Data Model (XDM) class that tracks the change history for a business account. This XDM class can only be included in the profile for customers with the B2B or B2P Edition.",
+  "definitions": {
+    "account-history": {
+      "properties": {
+        "xdm:accountEventKey": {
+          "title": "Account Event Key",
+          "description": "Unique composite key identifier for the Account Event.",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/b2b-source",
+          "meta:titleId": "account-history##xdm:accountKey##title##76911",
+          "meta:descriptionId": "account-history##xdm:accountKey##description##47311"
+        },
+        "xdm:accountKey": {
```

Comment:
Wouldn't we add "xdm:accountEventType" here as well?

Reply:
I added the accountEventType here as well